packages/react-core/src/components/DatePicker/__tests__/DatePicker.test.tsx

```diff
@@ -54,7 +54,7 @@ test('Error state can be cleared from outside', async () => {
 test('With popover opened', async () => {
   const user = userEvent.setup();
 
-  const { asFragment } = render(<DatePicker value="2020-03-18" />);
+  const { asFragment } = render(<DatePicker />);
```

Comment:
why are we removing the value here? is that not working?

Reply:
It was an attempt  to fix the test case that was failing in CI but passing locally for me.